src/components/social-media-component/assets/InstagramIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export default function InstagramIcon() {
```

Comment:
All icons should be stored in the same place so move the file to .src/icons, make it use also IconProps, remove <a> tag and use onClick with window.location.href